test/test_session.py

```diff
@@ -453,20 +482,32 @@ def new_file(session=None):
             grid_file.close()
 
         def find(session=None):
-            files = list(fs.find({"_id": 1}, session=session))
+            files = fs.find({"_id": 1}, session=session).to_list()
```

Comment:
I think this should still be `list` since it's a [GridOutCursor](https://pymongo.readthedocs.io/en/stable/api/gridfs/grid_file.html#gridfs.grid_file.GridOutCursor)